src/auth/user-record.ts

```diff
@@ -47,8 +47,13 @@ export interface MultiFactorInfoResponse {
   mfaEnrollmentId: string;
   displayName?: string;
   phoneInfo?: string;
+  totpInfo?: TotpInfoResponse;
   enrolledAt?: string;
-  [key: string]: any;
+  [key: string]: unknown;
+}
+
+export interface TotpInfoResponse {
+  [key: string]: unknown;
```

Comment:
Can we leave this empty for now? We can always add fields as they come up.

Reply:
We cannot declare an empty interface as per lint rules.